arcgis-ios-sdk-samples/Scenes/Show labels on layer in 3D/ShowLabelsOnLayer3DViewController.swift

```diff
@@ -33,9 +33,9 @@ class ShowLabelsOnLayer3DViewController: UIViewController {
         // Load the scene.
         scene.load { [weak self] error in
             guard let self = self else { return }
-            // Get the feature layer.
-            if let operationalLayers = scene.operationalLayers as? [AGSGroupLayer],
-               let groupLayer = operationalLayers.first(where: { $0.name == "Gas" }),
+            // Get the "Gas Main" feature layer.
+            let operationalLayers = scene.operationalLayers as [AnyObject]
+            if let groupLayer = operationalLayers.first(where: { $0.name == "Gas" }) as? AGSGroupLayer,
```

Comment:
```suggestion
            if let groupLayer = scene.operationalLayers.compactMap({ $0 as? AGSGroupLayer }).first(where: { $0.name == "Gas" })
```
Instead of casting the array to ` [AnyObject]`, I prefer first filter the elements down to `AGSGroupLayer`, then find the proper layer from it.